src/west/config.py

```diff
@@ -0,0 +1,56 @@
+'''
+Configuration file handling, using the standard configparser module.
+'''
+
+import configparser
+import os
+import platform
+
+from west.util import west_dir
+
+
+def read_config():
+    '''
+    Reads all configuration files, making the configuration values available as
+    a configparser.ConfigParser object in config.config. This object works
+    similarly to a dictionary: config.config['foo']['bar'] gets the value for
+    key 'bar' in section 'foo'.
+    '''
+
+    global config
+
+    #
+    # Gather (potential) configuration file paths, roughly following Git
+    # conventions.
+    #
+    # See the FILES section in git-config(1).
+    #
+
+    # System-wide
+
+    if platform.system() == 'Linux':
+        files = ['/etc/westconfig']
+    elif platform.system() == 'Darwin':  # Mac OS
+        # This was seen on a local machine ($(prefix) = /usr/local)
+        files = ['/usr/local/etc/westconfig']
+    elif platform.system() == 'Windows':
+        # Seen on a local machine. Not sure how it got derived though.
```

Comment:
What does "Not sure how it got derived though." mean?

Reply:
Derived by Git. See the `FILES` section in `git-config(1)`.

I'll remove it.